ophyd/areadetector/base.py

```diff
@@ -250,7 +250,9 @@ def visualize_asyn_digraph(self, *args, **kwargs):
         Parameters
         ----------
         *args, **kwargs : networkx.draw_networkx args and kwargs.
-            For the allowed args and kwargs see the `networkx.draw_networkx documentation <https://networkx.github.io/documentation/networkx-1.10/reference/generated/networkx.drawing.nx_pylab.draw_networkx.html>`
+            For the allowed args and kwargs see the `networkx.draw_networkx
```

Comment:
Looks like your editor "helpfully" broke this one line into three lines for you.